app/controllers/application_controller.rb

```diff
@@ -11,9 +12,10 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :null_session
 
+  before_filter :cors_preflight_check
```

Comment:
cors_preflight_check występuje także w supervisors_controller - w zamierzeniu tylko tamten controller miał na razie obsługiwać CORS. Myślę, że można to w takim razie stąd usunąć.